Makefile

```diff
@@ -79,16 +79,20 @@ help: ## Display this help.
 ##@ Development
 
 .PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+manifests: controller-gen crd ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) \
 		rbac:roleName=manager-role output:rbac:artifacts:config=config/components/rbac\
 		crd:generateEmbeddedObjectMeta=true output:crd:artifacts:config=config/components/crd/bases\
 		webhook output:webhook:artifacts:config=config/components/webhook\
-		paths="./..."
+		paths="{./api/..., ./pkg/...}"
+
+.PHONY: crd
+crd: kustomize # Write CRD to independent file used for validation.
```

Comment:
Do we have any protections to make sure that someone commits an update to the crd when bumping the api?

Reply:
I think we would actually. https://github.com/kubernetes-sigs/jobset/blob/9102d3f75b08a40457504350700c761c4702ef36/Makefile#L137

Reply:
Not yet, one thing I was thinking is we could add the path to the crd directory into those we perform git diffs on in the `verify` make rule

Reply:
I think its covered here actually.

`make verify` calls manifests generates and then checks the diff.

It looks like you added `make crd` to the manifest or generate step so I think this should be fine.

Reply:
Ah right, perfect